packages/dapp/components/layout/Inventory.tsx

```diff
@@ -50,7 +50,7 @@ const Inventory = () => {
 
   function showIfBalanceExists(key: keyof Balances | keyof typeof namedContracts, name: keyof typeof tokenSvg, id: string) {
     const usdcFix = function () {
-      if (key == "usdc") return 6;
+      if (key == "usdc" || key == "usdt") return 6;
```

Comment:
usdc/usdt decimals vary per chain for example the decimals of usdc is 18 on bnb chain. so it will not work on other chains.

Reply:
I had no idea but is the correct solution to check the amount of decimals then render that amount of decimals every time?

I also think that this solution is fine for our purposes because we don't plan to support bsc. 

Basically only possibly roll ups that settle on Ethereum.